src/components/KonnectorInstall.jsx

```diff
@@ -69,6 +68,7 @@ export const KonnectorInstall = ({
         {!success && (
           <AccountLoginForm
             connectorSlug={connector.slug}
+            connectorName={connector.name}
```

Comment:
Same here s/connectorName/konnectorName.

First rule of Cozy-Collect Legacy Club is "Let's keep the connector with a C for legacy code but let's use a K for brand new code and improvements" ✊